package.json

```diff
@@ -64,7 +64,7 @@
     "jest": "^29.4.3",
     "jest-environment-jsdom": "^29.4.3",
     "jest-environment-node-single-context": "^29.0.0",
-    "jest-runner-tsd": "^4.0.0",
+    "jest-runner-tsd": "^5.0.0",
```

Comment:
You can always use latest version, because `@tsd/typescript` is peer dependency. Makes it easy to test on any TS version and have latest bugs fixed.